### PR TITLE
Add Angular 'Template-driven forms' example

### PR DESCRIPTION
**Pull request by draconteus**
Solving issue #37

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -844,6 +844,40 @@ export class ReactiveFormComponent implements OnInit {
   }
 }
 ```
+#### Template-driven example
```
Comment (feimosi) [inline README.md]: `Template-driven forms example`

File: README.md [hunk 2]
```diff
@@ -844,6 +844,40 @@ export class ReactiveFormComponent implements OnInit {
...
+
+```ts
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'template-driven-form',
+  template: `
+    <div>
+        <form (ngSubmit)="onSubmit()" #templateDrivenForm="ngForm" novalidate>
```
Comment (feimosi) [inline README.md:894]: Could you name the variable simply `#form` for brevity?

File: README.md [hunk 3]
```diff
@@ -844,6 +844,40 @@ export class ReactiveFormComponent implements OnInit {
...
+        </div>
+        <button type="submit" [disabled]="!templateDrivenForm.form.valid">Submit</button>
+        </form>
+    </div>
+  `
+})
+
+export class TemplateDrivenFormComponent {
+  constructor () {
+    this.model = { name: '', email: '' };
```
Comment (feimosi) [inline README.md]: This could be written as a class field right?
```ts
export class TemplateDrivenFormComponent {
    model = { name: '', email: '' };
```

**Review by feimosi — CHANGES_REQUESTED** [review]
Thanks!

File: README.md (hunk 1 above)
Comment (draconteus) [reply README.md]: Oh, yeah. I rewrite it.

File: README.md (hunk 2 above)
Comment (draconteus) [reply README.md:894]: I used a more verbose naming, because the relationship with the submit button is more understandable (in L907).

**Comment by draconteus** [comment]
Done! If you understand my reasoning at link naming to the form.

**Comment by feimosi** [comment]
Alright, looks good 👍

**Pull request merged**